packages/skill-evolution/lib/skill_evolution_common.py

```
@@ -0,0 +1,707 @@
+
+
+# ---------------------------------------------------------------------------
+# スキル単位ロック（起動口の競合防止）
+# ---------------------------------------------------------------------------
+
+
+LOCK_TTL_SECONDS = 3600  # この時間を超えたロックは stale とみなす
+
+
+def _is_stale_lock(path: str) -> bool:
+    """ロックが stale（保持プロセス死亡 or TTL 超過）かを判定する。
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)